open_discussions/settings.py

```diff
@@ -830,3 +830,7 @@ def get_all_config_keys():
 # livestream API credentials
 LIVESTREAM_SECRET_KEY = get_string("LIVESTREAM_SECRET_KEY", None)
 LIVESTREAM_ACCOUNT_ID = get_string("LIVESTREAM_ACCOUNT_ID", None)
+
+# course catalog video etl settings
+YT_DEVELOPER_KEY = get_string("YT_DEVELOPER_KEY", None)
+VIDEOS_CONFIG_FILE_LOCATION = get_string("VIDEOS_CONFIG_FILE_LOCATION", None)
```

Comment:
Should've flagged this earlier, but could you refactor these so it's a little clearer from the name how they're used?

`YT_DEVELOPER_KEY` -> `YOUTUBE_DEVELOPER_KEY`
`VIDEOS_CONFIG_FILE_LOCATION` -> `YOUTUBE_CONFIG_FILE_LOCATION`